### ISSUE TITLE
Wight freeze time is too long if Brute is attacking.

### ISSUE THREAD

**Issue by BrickVoid** [LogicTechCorp/NetherEx-Archive#275]
I'm playing the DDSS modpack for 1.12.2 - the FINAL version of the modpack - and I don't know how to last against the Brute if a Wight freezes me. Edit: It looks like a Blaze with a cloak or shroud around it. Was that called a Blazing Juggernaut? Anyway, it's in this modpack and if some mob attacks me while I'm frozen I die and have to figure out how to get rid of the Wight problem.

Unfortunately the issue I am experiencing is that there is no documentation that I can find that is specific to this Minecraft version, or if it is available, Google just won't show it to me.

Is there anything that can block the Wight's freeze attack? If not, is there a way to configure it's attack to be shorter? 15 seconds is an eternity in Minecraft no matter which way you look at it, deadly to player if they do not understand the NetherEx hostile mobs.

The NetherEx mod seems very unfinished, if it expects the player to figure hard combat mechanics out without an in-game manual or up-to-date wiki to help.

**Comment by LogicTechCorp** [comment]
In the config you can decrease the "thawRarity" (potionEffect/freeze section) value which will make it more likely that you'll unfreeze before the effect ends and you can increase the "chargeCooldown" value (entity/brute section) which will lengthen the time between Brute attacks. 

You can also try updating to the latest versions of LibraryEx and NetherEx as the versions in the modpack are nearly three years old. The latest versions do have a bunch of changes and improvements, but I'm not sure if they'll be compatible with the modpack. If you do choose to update, be sure to make a backup of your world in case there are any issues.

**Comment by BrickVoid** [comment]
I tweaked down the Wight's Freeze attack to a value of 10, seems okay now to deal with it. Brute wasn't terribly difficult to deal with after that change, so thank you very much for the information. I have played in singleplayer in many modpacks and have had to either turn off or tone down some of the stupidly overpowered mobs in various modded Minecraft versions. Why does every mod author assume that the player can fight as fast and as hard as they can, or in their style, or whatever? Some modpacks, if it's not been disabled, I just will use Peaceful mode just to go and get items I need to progress. Otherwise, it's a stupidly hard slog trying to get my corpse back over and over again because I'm not capable of what the mod author programmed for their mobs.
